### DISCUSSION TITLE
Line smoothing

### DISCUSSION THREAD

**Discussion by Voltikor** [GSI-CS-CO/chart-fx#582] [Q&A]
Is there anyway for me to have a smoother line on a XYChart, more akin to JavaFX's default LineChart look?

**Comment by wirew0rm** [comment]
``` java
errorDataSetRenderer.setPolyLineStyle(LineStyle.BEZIER_CURVE);
```
will give you a smooth bezier curve. As commented at the option, this will cause significant computational overhead, you might want to disable it if you have many data points.

The above is applicable if you have very few datapoints. if the jitter is in your actual data, you might want to use a MathDataSet to apply a smoothing filter.

Hope this helps :)
